site/content/blog/2025-07-21-timing-rule-changes/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Timing Related Scan Rule Changes"
+summary: >
+  Scan rules related to timing attacks have been split ore renamed.
+images:
+- https://www.zaproxy.org/blog/2025-07-21-timing-rule-changes/something.png
+type: post
+tags:
+- blog
+- browser
+date: "2025-07-21"
+authors:
+- thorin
+---
+
+# Time Based Scan Rule Changes
+
+We have recently implemented a number of [requested](https://github.com/zaproxy/zap-extensions/pull/4316) changes with regard to scan rules which perform time based attacks.
+
+![ZAPbot with Edge browser icon](images/blog-edge.png)
+
+These changes were made for two reasons:
+1. It provides users with greater control in creating scan policies.
+2. The Time Based rules are, under some circumstances, potentially more false positive prone.
```

Comment:
This was the reason for the original changes, would be preferable to be more clear which ones are still more FP prone.

Reply:
I can drop the second point and just change it into a sentence about policy flexibility 🤷‍♂️

Admittedly I haven't done a ton of testing with the current iteration of the time checks. I know we feel it's better but are we confident it's fully addressed and we can distinguish system or network delay vs injection delay?

Reply:
The whole point does not need to be removed just clarified IMO. I edited the original comment to link to the improvements and that we should mention said improvements too.

Reply:
I doubt those will be improved in the near feature, they are not based on time functions but functions that might take some time to execute.